<!DOCTYPE html>
<html>
  <head>
    <title>Photos</title>
  </head>
  <body>
    <h3>simlate result
</h3>
    <h3>engine: phy
</h3>
    <h3>ode_language: PYTHON
</h3>
    <h3>selected: 41,42,43,
</h3>
    <h3>Traceback (most recent call last):
</h3>
    <h3>  File &quot;graph_simulator_electron.py&quot;, line 290, in &lt;module&gt;
</h3>
    <h3>    df = pd.read_excel(str(filename), sheet_name=&quot;input&quot;, engine='openpyxl' ,usecols=['time','C_t'])
</h3>
    <h3>  File &quot;c:\Users\arizona95\anaconda3\envs\tensorflow\lib\site-packages\pandas\util\_decorators.py&quot;, line 311, in wrapper
</h3>
    <h3>    return func(*args, **kwargs)
</h3>
    <h3>  File &quot;c:\Users\arizona95\anaconda3\envs\tensorflow\lib\site-packages\pandas\io\excel\_base.py&quot;, line 364, in read_excel
</h3>
    <h3>    io = ExcelFile(io, storage_options=storage_options, engine=engine)
</h3>
    <h3>  File &quot;c:\Users\arizona95\anaconda3\envs\tensorflow\lib\site-packages\pandas\io\excel\_base.py&quot;, line 1233, in __init__
</h3>
    <h3>    self._reader = self._engines[engine](self._io, storage_options=storage_options)
</h3>
    <h3>  File &quot;c:\Users\arizona95\anaconda3\envs\tensorflow\lib\site-packages\pandas\io\excel\_openpyxl.py&quot;, line 522, in __init__
</h3>
    <h3>    super().__init__(filepath_or_buffer, storage_options=storage_options)
</h3>
    <h3>  File &quot;c:\Users\arizona95\anaconda3\envs\tensorflow\lib\site-packages\pandas\io\excel\_base.py&quot;, line 411, in __init__
</h3>
    <h3>    filepath_or_buffer, &quot;rb&quot;, storage_options=storage_options, is_text=False
</h3>
    <h3>  File &quot;c:\Users\arizona95\anaconda3\envs\tensorflow\lib\site-packages\pandas\io\common.py&quot;, line 711, in get_handle
</h3>
    <h3>    handle = open(handle, ioargs.mode)
</h3>
    <h3>FileNotFoundError: [Errno 2] No such file or directory: ''
</h3>
    <h3>
</h3>
  </body>
</html>